spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/sql/TestTableEncryption.java

```diff
@@ -65,6 +66,15 @@ protected static Object[][] parameters() {
         SparkCatalogConfig.HIVE.catalogName(),
         SparkCatalogConfig.HIVE.implementation(),
         appendCatalogEncryptionProperties(SparkCatalogConfig.HIVE.properties())
+      },
+      {
+        SparkCatalogConfig.REST.catalogName(),
+        SparkCatalogConfig.REST.implementation(),
+        appendCatalogEncryptionProperties(
+            ImmutableMap.<String, String>builder()
+                .putAll(SparkCatalogConfig.REST.properties())
+                .put(CatalogProperties.URI, restCatalog.properties().get(CatalogProperties.URI))
+                .build())
       }
```

Comment:
For some reason, three tests run for each test but I've only specified two parameters here.